config/src/defaults.rs

```diff
@@ -60,6 +60,11 @@ pub trait Defaults {
         Duration::from_secs(30)
     }
 
+    /// Default period for discovering peers
+    fn connections_feeler_peers_period(&self) -> Duration {
+        Duration::from_secs(2)
```

Comment:
Any rationale behind the default value?

Reply:
It's probably too low?

Reply:
It has to be as soon as possible, but speaking with @girazoki it has to be similar to handshake time. So finally we speak to use about 1 or 2 seconds